utils/mock-data-loader/bss/deals/deals-to-submit-to-tfm.js

```diff
@@ -0,0 +1,242 @@
+const { nowTimestamp } = require('../dates');
+const MANDATORY_CRITERIA = require('../mandatoryCriteria');
+const { MAKER } = require('../../portal/roles');
+
+// de-structure and create a new array, so `sort` doesn't impact 'MANDATORY_CRITERIA'
+let mandatoryCriteria = [...MANDATORY_CRITERIA].sort((a, b) => (a.version > b.version ? 1 : -1));
+// get the latest mandatory criteria (sorted by version)
+mandatoryCriteria = mandatoryCriteria[mandatoryCriteria.length - 1];
```

Comment:
mandatoryCriterion to show it's single?

Reply:
These lines are at the top of each BSS deal - could probably do with a chat at some point because I'm slightly confused 😅

Reply:
Also going to make a helper to do this - DRY principles and all that